### ISSUE TITLE
Add textboxes

### ISSUE THREAD

**Issue by Rincewind6996** [scrivanolabs/ScrivanoForLinux#11]
I know, I know this is aimed for handwriting, but I still think that it could be sometimes nice to be able to type or copy text.

**Comment by scrivanolabs** [comment]
Yes, I agree that it would be nice to have text support. As you said, the focus of Scrivano is handwriting and so far I have heavily prioritised improving that aspect of the app. Text support is planned, but I'm not sure when it will happen (I did play with some prototype implementations in the past).

**Comment by Rincewind6996** [comment]
Yeah, okay, that's great. I will look forward for the next version and hope that you managed to implement it. Keep up the great work!

**Comment by iamworldian** [comment]
Hey @scrivanolabs , is it a open source project ?
How to contribute with it ?
